notebooks/simple-feature-transformations-explain-local.ipynb

```diff
@@ -0,0 +1,512 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Copyright (c) Microsoft Corporation. All rights reserved.\n",
+    "\n",
+    "Licensed under the MIT License."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Explain binary classification model predictions with raw feature transformations\n",
+    "_**This notebook showcases how to use the interpret-community repo to help interpret and visualize predictions of a binary classification model.**_\n",
+    "\n",
+    "\n",
+    "## Table of Contents\n",
+    "\n",
+    "1. [Introduction](#Introduction)\n",
+    "1. [Project](#Project)\n",
+    "1. [Setup](#Setup)\n",
+    "1. [Run model explainer locally at training time](#Explain)\n",
+    "    1. Apply feature transformations\n",
+    "    1. Train a binary classification model\n",
+    "    1. Explain the model on raw features\n",
+    "        1. Generate global explanations\n",
+    "        1. Generate local explanations\n",
+    "1. [Visualize results](#Visualize)\n",
+    "1. [Next steps](#Next%20steps)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id='Introduction'></a>\n",
+    "## 1. Introduction\n",
+    "\n",
+    "This notebook illustrates how to use interpret-community to help interpret a binary classification model using an IBM dataset with employee attrition data. It uses one-to-one and one-to-many feature transformations from the raw data to engineered features. The one-to-many feature transformations include one hot encoding for categorical features. The one-to-one feature transformations apply standard scaling on numeric features. The interpret-community tabular data explainer in this repo is then used to obtain raw feature importances.\n",
+    "\n",
+    "Three tabular data explainers are demonstrated: \n",
+    "- TabularExplainer (SHAP)\n",
+    "- MimicExplainer (global surrogate)\n",
+    "- PFIExplainer\n",
+    "\n",
+    "| ![Interpretability Toolkit Architecture](./img/interpretability-architecture.png) |\n",
+    "|:--:|\n",
+    "| *Interpretability Toolkit Architecture* |\n",
+    "\n",
+    "<a id='Project'></a>       \n",
+    "## 2. Project\n",
+    "\n",
+    "The goal of this project is to classify employee attrition with scikit-learn and locally running the model explainer: \n",
+    "\n",
+    "1. Transform raw features to engineered features\n",
+    "2. Train a SVC classification model using Scikit-learn\n",
+    "3. Run 'explain_model' globally and locally with full dataset in local mode, which doesn't contact any Azure services.\n",
```

Comment:
Run global and local explanation with the full dataset in local mode, which doesn't contact any cloud services. (Probably best to avoid mentioning Azure here specifically)

Reply:
removed Azure mention.